thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/pojo/MergedAnomalyResultBean.java

```diff
@@ -107,7 +107,10 @@ public void setCollection(String collection) {
     this.collection = collection;
   }
 
-
+  /**
```

Comment:
Here, you see severity, later you do return weight. It's still confusing. Can you add more comments here for better clarification.